syntax = "proto3";

import "google/protobuf/timestamp.proto";
import "util.proto";

package runner;

service Runner {
  rpc RunTest (RunTestRequest) returns (RunTestResponse) {}
}

message RunTestRequest {
  string series_id = 1;
  google.protobuf.Timestamp time = 2;
  string test = 3;
}

message RunTestResponse {
  util.Flag flag = 1;
}
